packages/adapter-better-sqlite3/src/better-sqlite3.ts

```diff
@@ -200,8 +200,8 @@ export class PrismaBetterSQLite3Adapter extends BetterSQLite3Queryable<StdClient
 }
 
 type BetterSQLite3InputParams = BetterSQLite3Options & {
-  url: ':memory:' | (string & {})
-  shadowDatabaseURL?: ':memory:' | (string & {})
+  url: string & {}
+  shadowDatabaseURL?: string & {}
```

Comment:
```suggestion
  url: ':memory:' | (string & {})
  shadowDatabaseURL?: ':memory:' | (string & {})
```

Reply:
Where does the requirement for this type come from?

It may well receive `file::memory:` in practice.